### DISCUSSION TITLE
Options for smaller post title fonts

### DISCUSSION THREAD

**Discussion by willhansen** [Docile-Alligator/Infinity-For-Reddit#1005] [Completed Ideas] [answered]
When I'm scrolling through post titles, I want more posts visible in the screen at one time, but the post title fonts only go down to "small".  We could have some "extra small" and "extra extra small" options?

**Comment by Docile-Alligator** [comment]
Maybe.

**Comment by willhansen** [comment]
Still want

**Comment by Docile-Alligator** [comment, marked as answer]
It's available in v5.3.0-beta1 now.
